GDAX/AuthenticatedClient.py

```diff
@@ -56,16 +56,20 @@ def holdsPagination(self, accountId, list, after):
             self.holdsPagination(accountId, list, r.headers["cb-after"])
         return list
 
-    def buy(self, buyParams):
-        buyParams["side"] = "buy"
-        if not buyParams["product_id"]:
-            buyParams["product_id"] = self.productId
-        r = requests.post(self.url + '/orders', data=json.dumps(buyParams), auth=self.auth)
+    def buy(self, **kwargs):
+        kwargs["side"] = "buy"
+        if not "product_id" in kwargs:
+            kwargs["product_id"] = self.productId
```

Comment:
Why do we specify a default "product_id" in `buy` but not `sell`?

Would it make sense to combine these into a `_place_order` method that accepts a `side` argument? We could then make `buy` and `sell` available as [partialmethods](https://docs.python.org/3.6/library/functools.html#functools.partialmethod).